src/app/pages/MediaArticlePage/PagePromoSections/LatestMediaSection/LatestMediaItem/index.tsx

```diff
@@ -26,16 +28,18 @@ const LatestMediaItem = forwardRef<HTMLDivElement, LatestMediaItemProp>(
           className="removeBackground"
           css={styles.promoStyle}
         >
-          <div css={styles.imageWrapper}>
-            <Promo.Image
-              src={src}
-              altText={item.imageAlt ?? 'Media image placeholder'}
-              width={240}
-              height={135}
-            />
-            <LatestMediaIndicator duration={item.duration} />
-          </div>
-          <div css={styles.textWrapper}>
+          {!isLite && (
+            <div css={styles.imageWrapper}>
+              <Promo.Image
+                src={src}
+                altText={item.imageAlt ?? 'Media image placeholder'}
+                width={240}
+                height={135}
+              />
+              <LatestMediaIndicator duration={item.duration} />
+            </div>
+          )}
+          <div css={[styles.textWrapper, isLite && { width: '100%' }]}>
```

Comment:
When it's a media promo we pass the media type and duration to the heading/link for screen reader users, have we considered the UX for this? I assume video/podcast pages etc are available on .lite and we show a message informing users why there is no video/podcast etc?

Reply:
Also think you would want to show the media indicator visually perhaps?

Reply:
Yea we will definitely need some kinda of banner (maybe with a link to the canonical version?) on the likes of MAP/Podcast pages to say "go here to watch this content" or something.

Yea you're right, I'll look into showing the media indicator in this case, I was a bit heavy handed by hiding the whole thing here!